packages/cmrjs/cmr-utils.js

```diff
@@ -19,6 +19,20 @@ const { omit } = require('@cumulus/common/util');
 const { CMR } = require('./cmr');
 const { getUrl, xmlParseOptions } = require('./utils');
 
+function getS3KeyOfFile(file) {
```

Comment:
Is this different than https://github.com/nasa/cumulus/pull/802/files#diff-ca3cefbb14126111676e7960c9a0a835R5?